File: sdk/android/sqljson/DbQuerySelect.java
package ardvro.component.sqljson;

import java.io.Serializable;

import ardvro.core.connection.IConnection;

public class DbQuerySelect extends DbQuery implements Serializable
{
    public String Fields = "*";

    public DbQuerySelect(DbQuery query, String fields, IConnection connection)
    {
        super(connection);
        Base = query;
        Connection = connection;
        QueryType = "SELECT";
        Fields = fields;
    }



}

File: sdk/.net/ardvro.sdk.net/component/sqljson/db/DbQuerySelect.cs
namespace ardvro.sdk.net.component.sqljson.db;

using ardvro.core.ext.connection;

[Serializable]
public class DbQuerySelect : DbQuery
{
    public string Fields = "*";

    public DbQuerySelect(DbQuery query, String fields, IConnection connection)
        :base(connection)
    {
        Base = query;
        Connection = connection;
        QueryType = "SELECT";
        Fields = fields;
    }


}
